src/ccmain/tesseractclass.cpp

```diff
@@ -619,6 +619,12 @@ void Tesseract::SetBlackAndWhitelist() {
   unicharset.set_black_and_whitelist(tessedit_char_blacklist.string(),
                                      tessedit_char_whitelist.string(),
                                      tessedit_char_unblacklist.string());
+  if (lstm_recognizer_) {
```

Comment:
Similar code might be needed in the `for` loop below (after line 633) for the `sub_langs_`.

Reply:
sure, I will have a look.

Reply:
done